src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/SchemaFormStore.js

```diff
@@ -0,0 +1,169 @@
+// @flow
+import {action, computed, observable} from 'mobx';
+import type {FormStoreInterface, RawSchema, Schema, SchemaEntry} from '../types';
+import metadataStore from './metadataStore';
+
+export default class SchemaFormStore implements FormStoreInterface {
```

Comment:
not sure about this name as it is very similar to `MemoryFormStore` and `ResourceFormStore` but the class is quite different from them. but i dont really have a better suggestion 🙈 is it common to mark decorator classes with a suffix or something?

Reply:
I think I have seen examples without a decorator suffix and with a decorator suffix. `SchemaFormStoreDecorator` would also be fine for me, especially because nobody will instantiate that class themselves anyway :slightly_smiling_face:

Reply:
would like `SchemaFormStoreDecorator` better because it hints that it is not another `FormStore` implementation 🙂